app/models/organization/creator.rb

```diff
@@ -162,7 +172,10 @@ def title
 
       github_client = users.sample.github_client
       github_org = GitHubOrganization.new(github_client, github_id)
-      github_org.name.present? ? github_org.name : github_org.login
+
+      org_identifier = github_org.name.presence || github_org.login
+
+      find_unique_title("#{org_identifier}-classroom-1")
```

Comment:
Should this number increment based on the number of classrooms in a given organization?

Reply:
The postfix gets incremented by `find_unique_title` if it already exists